buildchain/buildchain/salt_tree.py

```diff
@@ -216,9 +216,15 @@ def _get_parts(self) -> Iterator[str]:
         renderer=targets.Renderer.JSON,
     ),
 
+    Path('salt/metalk8s/addons/dex/ca/init.sls'),
+    Path('salt/metalk8s/addons/dex/ca/installed.sls'),
+    Path('salt/metalk8s/addons/dex/ca/advertised.sls'),
+    Path('salt/metalk8s/addons/dex/certs/init.sls'),
+    Path('salt/metalk8s/addons/dex/certs/server.sls'),
```

Comment:
Dex CA and server certificates are actually part of the Dex deployment, should be squashed with the chart render commit.

However, configuring k-a's OIDC plugin to point to Ingress, that **is** a separate commit.